react/index.d.ts

```diff
@@ -270,9 +276,9 @@ declare namespace React {
     //
     // Event System
     // ----------------------------------------------------------------------
-    interface SyntheticEventBase<CURRENT, TARGET> {
+    interface SyntheticEventBase<Current, Target> {
```

Comment:
Sorry for coming in late to the discussion. Correct me if I'm wrong, but it seems that we can't do better than `EventTarget` for the `target` property. If that's true, why do we need a second type parameter for it?

Can we use the following definition, for exampe?

```ts
interface SyntheticEvent<T extends EventTarget> {
    ...
    currentTarget: T;
    ...
    target: EventTarget;
}
```